ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -383,7 +384,7 @@ pub(crate) struct Module {
     /// The name of the module and the eventual symbol name for the JITted code.
     name: String,
     /// The IR trace as a linear sequence of instructions.
-    instrs: Vec<Instruction>,
+    instrs: TraceVec,
```

Comment:
Should this be `InstrVec` or `VecInstrs` or similar?

Reply:
Either sounds fine to me. Maybe `InstrVec` is slightly better?

Reply:
I prefer `InstrVec`